client.go

```diff
@@ -437,6 +458,11 @@ func (c *TunaSessionClient) listenNet(i int) {
 
 			c.handleConn(conn, sessKey, i)
 		}(conn)
+
+		<-done
```

Comment:
You can no longer accept concurrent connections

Reply:
Yes, modified.